packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -63,9 +101,34 @@ export default class metadataBackup {
     removeJob: (id: string) => Promise<void>,
   }
 
-  constructor(app: any) {
+  get runningMetadataRestores() {
+    return this._runningMetadataRestores
+  }
+
+  constructor(app: any, { remoteOptions }) {
     this._app = app
-    app.on('start', () => {
+    this._logger = undefined
+    this._runningMetadataRestores = new Set()
+
+    let debounceDelay = Number(remoteOptions.fetchBackupsDelay)
+    debounceDelay = Number.isNaN(debounceDelay)
+      ? DEFAULT_DEBOUNCE_DELAY
+      : debounceDelay
+
+    this._listXoMetadataBackups = debounceWithKey(
```

Comment:
You can still use the class properties syntax, but if you don't want to, remove the Babel conf for it.

Reply:
The construction is loaded after the decorator definition.

Reply:
Then remove the now unnecessary Babel config :slightly_smiling_face: